README.md

```diff
@@ -2,45 +2,153 @@
 
 [![REUSE status](https://api.reuse.software/badge/github.com/open-component-model/ocm-controller)](https://api.reuse.software/info/github.com/open-component-model/ocm-controller)
 
-Main repository for ocm-controller.
+This is the main repository for `ocm-controller`. The `ocm-controller` is designed to enable the automated deployment of software using the Open Component Model and Flux.
+
+It has the following features:
+- Resolves [`ComponentDescriptor`](https://github.com/open-component-model/ocm-spec/blob/main/doc/glossary.md#component-descriptor) metadata for a particular [component version](https://github.com/open-component-model/ocm-spec/blob/main/doc/glossary.md#component-version)
+- Performs authentication with [OCM repositories](https://github.com/open-component-model/ocm-spec/blob/main/doc/glossary.md#component-repository)
+- Retrieves [artifacts](https://github.com/open-component-model/ocm-spec/blob/main/doc/glossary.md#artifact) from OCM repositories
+- Verifies components
+- Makes individual resources from components available within the cluster
+- Performs localization and configuration of component resources
+
+## Quick Start
+
+### Pre-requisites
+- Create a kind cluster: `kind create cluster`
+- Make sure flux is installed in your cluster using: `flux install`
+- Install the controller using: `make install & make deploy`
+
+---
+
+In this tutorial we'll deploy the `phoban.io/podinfo` component which contains a Kubernetes `Deployment` manifest for the `podinfo` application.
```

Comment:
Should we push a version of podinfo under `ghcr.io/open-component-model/podinfo`?

Reply:
yes! I'll do that later today if I can